README.md

```diff
@@ -109,7 +115,7 @@ Write this grammar using EBNF with common extensions
 
 ## A2.3
 
-«replace this with your answer»
+IMG_4492.JPG
```

Comment:
# -2

There's a transition labelled "" going from S1 to S2. The trouble is that this always matches, so it makes the handling of S1 ambiguous: you're in S1 and the next word is "smelly". Where do you go? You could take the transition back to S1, or you could say "I'm not going to consume a word and go to S2".

You just need to merge S1 and S2 to fix this.